facilitator/src/gcp_oauth.rs

```diff
@@ -1,35 +1,59 @@
 use anyhow::{anyhow, Context, Result};
 use chrono::{prelude::Utc, DateTime, Duration};
+use dyn_clone::DynClone;
```

Comment:
This is a really cool library

Reply:
Yes, I was really glad it exists! Makes me think that maybe there is a missing language feature here, if `DynClone` can be implemented for arbitrary traits.